sap-2-tier-marketplace-image/shared/nic-config.json

```diff
@@ -31,12 +31,12 @@
       }
     },
     "newOrExistingSubnet": {
```

Comment:
This still seems like an odd pattern to me but I think we're stuck with it until we get if/then support - a lot of misdirection :)